anna/main.go

```diff
@@ -29,10 +29,11 @@ var (
 // Config represents the configuration used to create a new anna object.
 type Config struct {
 	// Dependencies.
-	Network spec.Network
-	Log     spec.Log
-	Server  spec.Server
-	Storage spec.Storage
+	FeatureStorage spec.Storage
+	GeneralStorage spec.Storage
```

Comment:
We should go one step back and provide a `Storages` interface which returns several storages. That way we create a nice API between provider and user. In case there will be yet another storage we only need to add it to the `Storages` interface. Then there is no need to change all implementations within the code base.